### PR TITLE
br tags for lines on divs

### PR DESCRIPTION
**Pull request by vv-monsalve**
Under the `div` elements a `<br>` tag is needed to have the blank lines.
They were added to each blank line within on divs (apart from ul's which add an space before and after by default)

### CODE REVIEW

File: gf-guide/license.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 <div class="callout">
 
 🐰 Google Fonts used to accept open-source licenses such as <mark class="grey">OFL</mark>, <mark class="grey">UFL</mark> and <mark class="grey">APACHE2</mark>. It has been a few years now that only OFL licenses are accepted.
-
+<bre><br>
```
Comment (RosaWagner) [inline gf-guide/license.md:9]: bre

Comment (vv-monsalve) [reply gf-guide/license.md]: `bre -> br` commit solves it

**Comment by vv-monsalve** [comment]
@RosaWagner I didn't want to merge this issue until you saw what it changes and why it's needed.

I think it's ready to be merged.

**Comment by vv-monsalve** [comment]
Oh boy, I can tell that about previews!

**Pull request merged**